src/Service.Host/client/src/components/POReqs/POReqUtility.js

```diff
@@ -289,7 +289,8 @@ function POReqUtility({ creating }) {
         !req.currency?.code?.length ||
         !req.country?.countryCode?.length ||
         !req.nominal?.nominalCode.length ||
-        !req.department?.departmentCode.length;
+        !req.department?.departmentCode.length ||
+        !req.dateRequired.length;
```

Comment:
can this explode if dateRequired is null or undefined?

Reply:
good point. will at least need the ? same as the others